spec/requests/admin/notify_downloads_spec.rb

```diff
@@ -0,0 +1,113 @@
+require 'rails_helper'
+
+RSpec.describe 'Admin Notify Downloads', type: :request do
+  include SingleSignOnHelpers
+
+  before do
+    stub_govuk_bank_holidays_request
+    mock_sso_with(email: 'admin@example.com')
+    get '/auth/google_oauth2/callback'
+  end
+
+  describe '#index' do
+    it 'lists the available Notify downloads' do
+      get admin_notify_downloads_path
+
+      expect(response).to be_successful
+      expect(response.body).to include('Notify downloads')
```

Comment:
Two small things: is there a reason you're using `include` and not `have_content`? Another even smaller one, you're putting the include parameter in brackets here, and not lower down the file.

Reply:
`have_content` is a Capybara `page` object thing and these are request specs, where you get an `ActionDispach::TestResponse` instead.